pkg/cloudprovider/providers/azure/azure_vmss.go

```diff
@@ -212,6 +212,8 @@ func (ss *scaleSet) GetInstanceTypeByNodeName(name string) (string, error) {
 }
 
 // GetZoneByNodeName gets cloudprovider.Zone by node name.
+// TODO(feiskyer): Add availability zone support of VirtualMachineScaleSetVM
+// after it is supported in Azure Go SDK.
```

Comment:
Note: this is used in cloud-controller-manager. It is waiting for new Azure Go SDK releases.